src/arm/bbb-bone-buses.dtsi

```diff
@@ -455,3 +455,20 @@ bone_spi_1: &spi1 {
 		spi-max-frequency = <16000000>;
 	};
 };
+
+// PWMs & TIMERs
+bone_pwm_1: &ehrpwm1 {
+
+};
+
+bone_pwmss_1: &epwmss1 {
+
+};
+
+bone_pwm_2: &ehrpwm2 {
+
+};
+
+bone_pwmss_2: &epwmss2 {
+
+};
```

Comment:
@jadonk @RobertCNelson I am not able to use symlinks with these nodes what could be the reason for that?
These nodes are used in https://github.com/beagleboard/bb.org-overlays/pull/188 and I will also use them for 4D systems LCD cape DT overlay.

Reply:
Is there even a udev rule for bone-style symlinks on pwms?

https://elinux.org/Beagleboard:BeagleBone_cape_interface_spec section on methodology need to be brought up-to-date on what is in https://github.com/beagleboard/customizations/ and planned to be in there.

Reply:
Hmmm.... I guess even just https://github.com/beagleboard/customizations/blob/master/etc/udev/rules.d/10-of-symlink.rules *should* cover it. What is our method for debugging udev rules? Does the rule ever run on these? Why doesn't the pull-request include the symlink definitions?